.vib/phpbb/goss/vars.yaml

```diff
@@ -0,0 +1,33 @@
+binaries:
+  - apachectl
+  - mariadb
+  - php
+  - render-template
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/phpbb
+      - /opt/bitnami/phpbb
+      - /opt/bitnami/phpbb/cache
+      - /opt/bitnami/phpbb/files
+      - /opt/bitnami/phpbb/images
+      - /opt/bitnami/phpbb/store
+      - /opt/bitnami/phpbb/ext
```

Comment:
The `configure_permissions_ownership` function applies the permissions **recursively**. And since these folders were already being checked in our previous tests, I considered appropriate adding them here.